packages/api/src/middleware/geolocate.js

```diff
@@ -58,7 +56,7 @@ function geoLocateFactory({ first = true, region = "region" }) {
     let data;
     try {
       if (first) {
-        data = await promiseAny(promises);
+        data = await Promise.any(promises);
```

Comment:
Do we actually use this on the API as well? Thought it was a worker thing only

Reply:
The functionality is still there, maybe we'd want to use it to geolocate the second-nearest entrypoint or something. Butyeah.